manual/manual/tutorials/coreexamples.etex

```diff
@@ -677,6 +677,94 @@ print_expr e; print_newline ();;
 print_expr (deriv e "x"); print_newline ();;
 \end{caml_example}
 
+\section{Printf formats}
+
+There is a "printf" function in
+\href{https://caml.inria.fr/pub/docs/manual-ocaml/libref/Printf.html}{Printf}
+module (see chapter~\ref{c:moduleexamples}) that allows you to make formatted
+output more concisely.
+It follows the behavior of the "printf" function from the C standard library.
+The "printf" function takes a format string that describes the desired output
+as a text interspered with specifiers (for instance "%d", "%f").
+Next, the specifiers are substituted by the corresponding arguments in the
```

Comment:
The term `corresponding` arguments may be unclear. A reader might think that it mean that

```OCaml
Printf.printf "%d + %B" true 1 
```
works because `true` is the corresponding argument for `%B`.
Maybe
> the following arguments in their order of apparition

?